gallery/visualtests/nxCollapsibleMultiSelect.js

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
A bunch of files from the wrong branch here

Reply:
> A bunch of files from the wrong branch here

Removed: b3d6904805a123dc40fb450454f31f184968170a